abci/example/kvstore/kvstore.go

```diff
@@ -68,13 +68,20 @@ type Application struct {
 
 	state        State
 	RetainBlocks int64 // blocks to retain after commit (via ResponseCommit.RetainHeight)
+	// If true, the app will generate block events in BeginBlock. Used to test the event indexer
+	// Should be false by default to avoid generating too much data.
+	genBlockEvents bool
 }
 
 func NewApplication() *Application {
 	state := loadState(dbm.NewMemDB())
 	return &Application{state: state}
 }
 
+func (app *Application) SetGenBlockEvents() {
```

Comment:
Is this method ever used anywhere?

Reply:
This is not used atm but should be used if we want to turn on the generation of Blockevents.